crowdsourcing/models.py

```diff
@@ -274,7 +274,7 @@ class TemplateItem(models.Model):
     data_source = models.CharField(max_length=256, null=True)
     layout = models.CharField(max_length=16, default='column')
     type = models.CharField(max_length=16)
-    sub_type = models.CharField(max_length=16)
```

Comment:
I would lean towards adding new component - number than adding specific details to input component. Number can be integer, decimal, range etc. so better to keep it separate.